server/src/site/xdoc/upgrade-2.3.xml

```diff
@@ -55,6 +55,18 @@
          
     </subsection>
 
+    <subsection name="Limitations">
+         
+     <p>The migration is only available for the same backend.
```

Comment:
What about 2.2.3 => 3.0 ? Are you sure JPA tables did not change?

There's a possibility to launch james with 2 mbx managers, register them, and do copy accross stores. I never tested it however, making it working should not be a big deal (even if it might involve running a custom James :trollface: )

The migration on same backend from 2.3 to 3.0 is not supported at all whatever backend, I believe.